hydra-cluster/src/CardanoClient.hs

```diff
@@ -169,19 +167,20 @@ mkGenesisTx networkId pparams signingKey initialAmount recipients =
 data RunningNode = RunningNode
   { nodeSocket :: SocketPath
   , networkId :: NetworkId
-  , pparams :: PParams Api.LedgerEra
+  , blockTime :: DiffTime
+  -- ^ Expected time between blocks (varies a lot on testnets)
   }
 
 -- Logging
 
 data NodeLog
-  = MsgNodeCmdSpec Text
+  = MsgNodeCmdSpec {cmd :: Text}
   | MsgCLI [Text]
   | MsgCLIStatus Text Text
   | MsgCLIRetry Text
   | MsgCLIRetryResult Text Int
   | MsgNodeStarting {stateDirectory :: FilePath}
-  | MsgSocketIsReady FilePath
+  | MsgSocketIsReady SocketPath
```

Comment:
why this change if `unFile SocketPath` returns FilePath?

is it because SocketPath displays better?

Reply:
I chose it because its a "stronger" type.